Source/WebInspectorUI/UserInterface/Views/DataGrid.css

```diff
@@ -92,6 +95,7 @@
 .data-grid th > .header-cell-content {
     margin: 3px 0;
     padding: 0 var(--data-grid-header-horizontal-padding);
+    line-height: 1.5;
```

Comment:
> this appears to slightly increase the height of dividers between header cells

Yes, that is an acknowledged side-effect of the fact that the dividers are the border of the block element inside the table cell:

```
.data-grid th:not(:last-child) > .header-cell-content {
    border-inline-end: 1px solid var(--border-color);
}
```

We don't have control for border height in CSS yet. The optical change of the divider height is minimal and didn't warrant an hackier implementation to preserve it.

> vertically shifts the content in header cells (even for english text)

That is the effect of the increased line height to accommodate fonts with longer descenders or ascenders.
But even in English, the text remains vertically aligned to baseline. Here's a example of an extreme `line-height: 10`:

<img width="1282" alt="Screenshot 2024-05-03 at 11 11 36" src="https://github.com/WebKit/WebKit/assets/105809798/c54e79b5-a28d-4186-9d43-e6783579e5b5">

Reply:
is there any way to not increase the height of the dividers?  maybe something as simple as `margin-block: 1px`?  i ask because i think this is different from macOS style which we've followed (as best we can)

Reply:
We can add `padding: 0;` and `margin: 0;` to `.data-grid th:not(:last-child) > .header-cell-content`, `line-height: 1.5;` to `.data-grid th > .header-cell-content`, and `margin-block: 1px; padding: 0;` to `.data-grid.no-header > .header-wrapper` to not increase height of the dividers.